base/bucket_gocb.go

```diff
@@ -1421,8 +1443,13 @@ func (bucket *CouchbaseBucketGoCB) GetXattr(k string, xattrKey string, xv interf
 		}
 
 	}
-	description := fmt.Sprintf("GetXattr %s", UD(k).Redact())
-	err, result := RetryLoop(description, worker, bucket.spec.RetrySleeper())
+
+	err, result := RetryLoop("GetXattr", worker, bucket.spec.RetrySleeper())
+	if err != nil {
+		if _, ok := err.(*RetryTimeoutError); ok {
```

Comment:
Also here - I think we still need to handle all types of error. Not just timeout.

Reply:
Same as other one